### PR TITLE
fix/feature: Selecting the save button in Center Settings now properly saves information.  Added demote button to Stewards tab to demote stewards.

### PR DESCRIPTION
**Pull request by J-Fid**
Please Review:

@mannyrivera2010 
@blynch11p 
@skhtet 
@mleeBoeing 

To test saving info in Center Settings:
* Go to Center Settings
* Edit content in any of the tabs
* Select save
* Observe that changes were saved

To test demotion of an Org Steward:
* Navigate to Center Settigns -> Stewards
* Select a steward
* Select the Demote button and attempt to remove the steward
* Observe successful removal of steward
* Observe the grid refreshes with removed steward no longer appearing

Error handling:
* Attempt to select the Demote button without selecting a steward
* Observe error message appears
* Attempt to demote the last steward in the list
* Observe error message appears

### CODE REVIEW

**Comment by blynch11p** [comment]
Functionally, this looks good on initial testing, but a couple comments for consistency usage throughout the app:

1: Clicking 'Demote' when a row is highlighted:
  We use a styled confirmation dialog box for removing apps.  We should use a similar dialog box for remove Stewards.
screenshot from 2017-06-26 16-26-31

2. Disable the 'Demote' button when no rows are selected, much like the edit button.  

2b. Disable the 'Demote' button when only one row is left in the table, regardless of if the row is selected.

3. The 'Edit' button has tooltip text over it ('Edit Selected Record').  Add similar text to the Demote button.

File: app/js/components/management/mall/Stewards.jsx [hunk 1]
```diff
@@ -59,6 +63,53 @@ var Stewards = React.createClass({
...
+                                    .yes(function () {
+                                        if (newGrid.records.length <= 1){
+                                            w2alert('There must be at least one Steward.');
+                                        }
+                                        else{
+                                            var newUserInfo = {"stewardedOrganizations": [],
+                                            "user":{"groups":[{"name":"USER"}]}
+                                            };
+
+                                            $.ajax({
```
Comment (mleeBoeing) [inline app/js/components/management/mall/Stewards.jsx]: Should move this to ozp-center/app/js/webapi/Profile.js

**Comment by mleeBoeing** [comment]
+1 to what @blynch11p said. Nice job!

**Comment by J-Fid** [comment]
> 1: Clicking 'Demote' when a row is highlighted:
> We use a styled confirmation dialog box for removing apps. We should use a similar dialog box for removing Stewards.

This was moved to a new ticket.  Other button changes are working as intended.

**Comment by blynch11p** [comment]
Looks good.
Demote button removes Stewards as expected.
The Demote button is only active when a row is selected (and there is more than one row available.)
The Edit button still works as expected.

:+1:

**Comment by mleeBoeing** [comment]
:+1:

**Pull request merged**